src/panels/experimental-ui/hui-view.js

```diff
@@ -102,6 +102,10 @@ class HUIView extends PolymerElement {
         // eslint-disable-next-line
         console.error('Unknown type encountered:', cardConfig.type);
         continue;
+      } else if (!customElements.get(tag)) {
+        // eslint-disable-next-line
+        console.error('Custom element doesn\'t exist:', tag);
```

Comment:
This is ok. I wonder in a future PR if we should have an "error" card. One that renders the error message. `console.error` is not easily spotted by users.

Reply:
we can send it to our logger as frontend error?

Reply:
I think rendering a red card is more obvious, people are looking for their card and don't see it.

Wrote it up: https://github.com/home-assistant/ui-schema/issues/9